plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -565,6 +565,18 @@ public void testPathColumn()
         }
     }
 
+    @Test
+    public void testTableLocationTrailingSpace()
+    {
+        String tableName = "table_with_space ";
```

Comment:
I would recommend appending a space in location, not table name. 
```suggestion
        String tableName = "table_with_space";
```